hadoop-hdds/hadoop-dependency-test/pom.xml

```diff
@@ -54,6 +54,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+    </dependency>
```

Comment:
Is this change related?

Reply:
`OzoneBlockTokenIdentifier` is in `hadoop-hdds/common`, so I'm moving the test to the same submodule.  This dependency is required at runtime for Hadoop code invoked by the test.

```
[ERROR] Tests run: 4, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.506 s <<< FAILURE! - in org.apache.hadoop.hdds.security.token.TestOzoneBlockTokenIdentifier
[ERROR] org.apache.hadoop.hdds.security.token.TestOzoneBlockTokenIdentifier.testTokenSerialization  Time elapsed: 0.04 s  <<< ERROR!
java.lang.NoClassDefFoundError: org/apache/commons/codec/binary/Base64
	at org.apache.hadoop.security.token.Token.encodeWritable(Token.java:342)
	at org.apache.hadoop.security.token.Token.encodeToUrlString(Token.java:373)
	at org.apache.hadoop.hdds.security.token.TestOzoneBlockTokenIdentifier.testTokenSerialization(TestOzoneBlockTokenIdentifier.java:169)
```